harvard-leeds-beckett-university.csl

```diff
@@ -474,9 +474,7 @@
                 </if>
               </choose>
               <group delimiter=" ">
-                <names variable="author">
-                  <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
-                </names>
+                <names variable="author"/>
```

Comment:
@adam3smith I did consider this, though the documentation simply specifies, “The performers, followed by a full stop.” Make of that what you will.